tests/testthat/test-pstr_aggregate_scores.R

```diff
@@ -0,0 +1,21 @@
+test_that("FIXME: Address the warnings (#29)", {
+  companies |>
+    pstr_add_reductions(ep_weo, weo_2022) |>
+    pstr_add_transition_risk() |>
+    pstr_aggregate_scores() |>
+    # FIXME: Use multiple = "all"
+    expect_warning("Each row.*is expected to match at most 1 row") |>
+    # FIXME: Replace summarise() with reframe()
```

Comment:
Please review all calls to `summarize()` and check if instead you should use `reframe()`. This will solve some warnings, but will also confront you with deciding what to do with the groups. `reframe()` always drops groups, so if you have a good reason to keep gropus, you'll need to add them again after calling `summarize()`.

Reply:
It turned out to be a lil tricky and maybe there is a bug in `dplyr::left_join()` so I went ahead and did it myself to learn a bit more about the issue and to save you time you can use for other more straightforward tasks.